src/GraphQL.Tests/Utilities/FederatedSchemaBuilderTests.cs

```diff
@@ -2,6 +2,7 @@
 using GraphQL.Utilities.Federation;
 using Xunit;
 using Newtonsoft.Json.Linq;
```

Comment:
remove?

Reply:
Looks like there's a JObject.Parse and then use of `SelectToken` in here that I'd have to rewrite. There's no SelectToken equiv in S.T.Json, but I might be able to just read it outta the strongly-typed `ExecutionResult` instead. Let me see.

Reply:
It's not pretty, but I've fixed this up. I've also removed all other uses of newtonsoft.json in other tests, the only last remaining usage is in the test harness middleware, which I'll convert over now since it's a 3.1 project.